src/SignalR/common/testassets/Tests.Utils/TaskExtensions.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Threading.Tasks
 #endif
     static class TaskExtensions
     {
-        private const int DefaultTimeout = 5000;
+        private const int DefaultTimeout = 30*1000;
```

Comment:
nit: Spacing..

Reply:
It would be great if we could capture a dump or get some data on when tests actually take > 5 seconds to time out.

Reply:
I can take a look at all the current flaky test failures we have and see which ones failed due to timeouts, but yeah more data on this would be helpful in making this decision